apps/server/src/modules/tool/controller/dto/request/external-tool-search.params.ts

```diff
@@ -6,4 +6,9 @@ export class ExternalToolSearchParams {
 	@IsString()
 	@IsOptional()
 	name?: string;
+
+	@ApiPropertyOptional()
+	@IsString()
+	@IsOptional()
+	clientId?: string;
```

Comment:
Is this the clientId of the external tool? Can you add a API decorator if small explanation?